test/node/parser/serializer.test.ts

```diff
@@ -88,5 +91,13 @@ describe('serialize()', () => {
       expect(() => BSON.serialize(Buffer.alloc(2))).to.throw(/cannot be BSON documents/);
       expect(() => BSON.serialize(new Uint8Array(3))).to.throw(/cannot be BSON documents/);
     });
+
+    it(`throws if Symbol.for('@@mdb.bson.version') is the wrong version`, () => {
```

Comment:
how about if the symbol is not defined? (as it presumably wouldn't be for older versions of bson)

Reply:
added case